### PR TITLE
Update matador dependencies

### PR DESCRIPTION
**Pull request by nicks**
Hello @kylehg, 

Please review the following commits I made in branch 'nick-connect'.

15e245585c45d8b813dc0647a330cadf30c029cb (2015-06-26 13:17:25 -0700)
Update matador dependencies

R=@kylehg

Test plan: 'not tested'

### CODE REVIEW

**Comment by kylehg** [comment]
lg

**Comment by nicks** [comment]
actually, on second thought, i'm going to try to update to connect 3. It looks like they have some security fixes that we want.
https://github.com/senchalabs/connect/wiki/Connect-3.0

**Comment by kylehg** [comment]
Ok, sgtm. Do you know why we're in `alpha` btw? And would bumping to connect 3.0 be breaking / require bumping our major version as well?

**Comment by nicks** [comment]
ok! all modules replaced with the newer versions. ptal?

File: package.json [hunk 1]
```diff
@@ -21,10 +21,16 @@
...
   },
   "dependencies": {
-    "connect": "2.3.3",
-    "valentine": "1.8.0",
+    "body-parser": "^1.13.1",
+    "connect": "^3.4.0",
+    "cookies": "^0.5.0",
+    "cookie-parser": "^1.3.5",
+    "errorhandler": "^1.4.0",
+    "qs": "3.1.0",
```
Comment (kylehg) [inline package.json]: Any reason not to use carets for `qs` and `valentine`? For consistency?

File: tests/support/functional.js [hunk 2]
```diff
@@ -13,10 +13,6 @@ var connect = require('connect')
 
 module.exports = request
 
-connect.proto.request = function(){
-  return request(this)
-}
-
```
Comment (kylehg) [inline tests/support/functional.js]: Hooray for not monkey-patching imported objects!

**Comment by kylehg** [comment]
lgtm

File: package.json (hunk 1 above)
Comment (nicks) [inline package.json]: i'm fine with `qs`, though i think we want to get rid of `valentine`

**Pull request merged**